core-meetings/2020-03-11-issue-management-meeting.md

```diff
@@ -0,0 +1,47 @@
+	- We asked for a Snack, even though the issue requires switching between development and release. The example code used to create a minimal repro for the Snack can be useful to whoever investigates the issue later.
+- **PropTypes to prop-types, Error evaluating React.PropTypes:** https://github.com/facebook/react-native/issues/28282
+    - The environment info for this issue shows React Native 0.55 being used, but 0.61 is the latest stable as of this writing on March 11, 2020. We ask that everyone try the latest version before filing an issue, in order to eliminate the possibility that the issue may have been solved already. 
+	- Note that caution must be taken when looking at older issues. The author may have indeed verified the issue on what happened to be the latest release when they originally filed the issue. 
+	- The react-native-bot will only comment on an issue at creation time if the version listed does not match latest in the releases tab on GitHub. The bot will re-check if the issue is edited, but only to remove the label. It will not remove its own comment if it finds the issue is now using the latest version.
+- **Metro bundler does not start:** https://github.com/facebook/react-native/issues/28281
+	- This was marked as a duplicate of a closed issue by someone in the thread.
+	- We looked at the closed issue, and found it was closed after someone suggested they should post it to the CLI repo (https://github.com/react-native-community/cli/issues/484).
+	- We let the author know that their issue was marked resolved there. If they need to, they can open a new issue in the cli repo.
+- **Android ignores unicode soft hyphen:** https://github.com/facebook/react-native/issues/28279
+	- This issue makes good use of images to display the problem.
+	- We labeled it as affecting the Text component, as well as being an issue that described a disparity in how different platforms behave.
+- **Crash in RCTWebSocketModule:** https://github.com/facebook/react-native/issues/28278
+	- Crash reports are not a good fit for the repository. We ask the authors to investigate further and determine what the minimal repro is that causes the crash. Without this, it's unlikely anyone can help the author ascertain what causes the crash as we don't have access to the app's source code or the repro steps. 
+	- Even if we did have the source code and repro steps, it's out of scope of the repo to investigate crashes in individual apps without a minimal repro that can be reproduced in a Snack or a brand new "rn init" app.
+- **StatusBar and Flexibility:** https://github.com/facebook/react-native/issues/28276
+	- Good use of <details> and images to demonstrate the issue. Repro steps describe how one might get to a repro, but we do need actual code (ideally a Snack). With a repository of this size, the bar is set so that the person reporting an issue is responsible for doing the work to provide the repro.
```

Comment:
Do you want to escape `<details>`? In the rich diff it looks like it hides the rest of this. Is that on purpose?

Reply:
Good catch!